lib/tarantool_server.py

```diff
@@ -886,6 +895,11 @@ def start(self, silent=True, wait=True, wait_load=True, rais=True, args=[],
                     color_stdout('\n[Instance "{0.name}"] Tarantool server '
                                  'failed to start\n'.format(self),
                                  schema='error')
+                    if err.timeout:
+                        color_stdout(
+                            '\n[Instance "{0.name}"] Tarantool server start '
+                            'timeout {1} was reached.\n'.format(
+                                self, err.timeout), schema='error')
```

Comment:
Can we tweak the line above to reuse `TarantoolStartError.__str__()` and, so, don't duplicate the error reporting code?

Reply:
Fixed